app/datamapper_setup.rb

```diff
@@ -0,0 +1,13 @@
+require 'data_mapper'
+
+env = ENV['RACK_ENV'] || 'development'
+
+DataMapper.setup(:default, ENV['DATABASE_URL'] ||
+ "postgres://localhost/chitter_challenge_#{env}")
+
+require_relative 'models/peep'
+require_relative 'models/user'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.